addons/website/views/snippets/snippets.xml

```diff
@@ -373,11 +373,10 @@
          data-selector="section"
          data-target="> .carousel">
         <we-select string="Style">
+            <we-button data-select-class="s_carousel_default">Default</we-button>
```

Comment:
Sorry for the delay reviewing this, but let's merge that part in master, not 14.0.
As you edit the commit, edit the commit message too. Your title is supposed to complete the sentence "when applied, this commit will ..." so "when applied, this commit will small fixes snippet carousel" does not mean anything ;)

Reply:
@qsm-odoo by " that part" you mean this line or this commit ?

Reply:
The CSS is for 14.0 yes. Only the XML change which is not a bug has to be for master